### PR TITLE
Run goreleaser in parallel

### PR DESCRIPTION
**Pull request by rvolosatovs**
#### Summary

Since https://github.com/goreleaser/goreleaser/issues/1120 is fixed in most recent `goreleaser` version,  d77b09b564bd50a0ffd8ff2cef48e4a8d062c0e8 can be reverted.

#### Changes

- Run `goreleaser` in parallel
- Update `goreleaser`

### CODE REVIEW

**Comment by rvolosatovs** [comment]
`go mod` fails with
```
go: github.com/go-macaron/cors@v0.0.0-20190309005821-6fd6a9bfe14e9: unexpected status (https://proxy.golang.org/github.com/go-macaron/cors/@v/v0.0.0-20190309005821-6fd6a9bfe14e9.info): 410 Gone
```
:thinking:

**Comment by rvolosatovs** [comment]
Refs https://github.com/goreleaser/goreleaser/issues/1132

File: go.mod [hunk 1]
```diff
@@ -39,14 +42,14 @@ require (
 	github.com/envoyproxy/protoc-gen-validate v0.2.0-java
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/getsentry/raven-go v0.2.0
-	github.com/go-redis/redis v0.0.0-20190503082931-75795aa4236d
```
Comment (htdvisser) [inline go.mod]: Can we leave this the way it was?

Comment (rvolosatovs) [reply go.mod]: This is caused by `go get github.com/goreleaser/goreleaser@v0.117.1`
I have no idea why, but I think in this case `go` knows better :man_shrugging:

**Pull request merged**